opl/skip_to_end.py

```diff
@@ -57,6 +76,7 @@ def doit(args, status_data):
         args.kafka_timeout,
         args.kafka_topic,
         args.kafka_group,
+        args,
```

Comment:
Change it just to send args then as we have all of the other stuff there anyway

Reply:
Good point!